spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/TracingProperties.java

```diff
@@ -53,4 +74,119 @@ public void setProbability(float probability) {
 
 	}
 
+	public static class Baggage {
+
+		/**
+		 * Enables correlating the baggage context with logging contexts.
+		 */
+		private boolean correlationEnabled = true;
+
+		/**
+		 * List of fields that should be propagated over the wire.
+		 */
+		private List<String> correlationFields = new ArrayList<>();
+
+		/**
+		 * List of fields that should be accessible within the JVM process but not propagated
+		 * over the wire.
+		 */
+		private List<String> localFields = new ArrayList<>();
+
+		/**
+		 * List of fields that are referenced the same in-process as it is on the wire. For
+		 * example, the field "x-vcap-request-id" would be set as-is including the prefix.
+		 */
+		private List<String> remoteFields = new ArrayList<>();
+
+		/**
+		 * List of fields that should automatically become tags.
+		 */
+		private List<String> tagFields = new ArrayList<>();
+
+		public boolean isCorrelationEnabled() {
+			return correlationEnabled;
+		}
+
+		public void setCorrelationEnabled(boolean correlationEnabled) {
+			this.correlationEnabled = correlationEnabled;
+		}
+
+		public List<String> getCorrelationFields() {
+			return correlationFields;
+		}
+
+		public void setCorrelationFields(List<String> correlationFields) {
+			this.correlationFields = correlationFields;
+		}
+
+		public List<String> getLocalFields() {
+			return localFields;
+		}
+
+		public void setLocalFields(List<String> localFields) {
+			this.localFields = localFields;
+		}
+
+		public List<String> getRemoteFields() {
+			return remoteFields;
+		}
+
+		public void setRemoteFields(List<String> remoteFields) {
+			this.remoteFields = remoteFields;
+		}
+
+		public List<String> getTagFields() {
+			return this.tagFields;
+		}
+
+		public void setTagFields(List<String> tagFields) {
+			this.tagFields = tagFields;
+		}
+
+	}
+
+	public static class Propagation {
+
+		/**
+		 * Tracing context propagation types.
+		 */
+		// TODO: Should this be the default?
+		private PropagationType type = PropagationType.B3;
+
+		public PropagationType getType() {
+			return this.type;
+		}
+
+		public void setType(PropagationType type) {
+			this.type = type;
+		}
+
+		enum PropagationType {
+			/**
+			 * AWS propagation type.
+			 */
+			AWS,
+
+			/**
+			 * B3 propagation type.
+			 */
+			B3,
+
+			/**
+			 * W3C propagation type.
+			 */
+			W3C,
+
+			/**
+			 * Jaeger propagation type.
+			 */
+			JAEGER, // TODO: Supported by OTel, not by Brave - my suggestion is not to support it
```

Comment:
We could either not support it or fail on startup if someone sets the value to `jaeger` but they're using Brave. I don't know the space well enough to have an opinion about which would be best. If you're suggesting that we don't support it, that's good enough for me and this could be removed.